code/modules/mob/living/carbon/alien/say.dm

```diff
@@ -15,11 +15,21 @@
 	if(copytext(message,1,2) == "*")
 		return emote(copytext(message,2))
 
+	var/needtohack = 1 //hacky way to get around stupid byond shit
 	var/datum/language/speaking = parse_language(message)
+
+	if(speaking == null)
+		var/quickcheck = get_default_language()
+		if(quickcheck == null)
+			speaking = all_languages["Xenomorph"] //this is the hackiest thing, no clue how to avoid it
+			needtohack = 0
+		else
+			speaking = get_default_language()
+			needtohack = 0
+
 	if(speaking)

```

Comment:
Well shit, I redid what didn't work before- and of course it works now
https://github.com/tigercat2000/Paradise/commit/a951afb6cfa368b38ed8db4907a98435f688d7da